README.md

```diff
@@ -1,2 +1,15 @@
 # project-template
-A project template to use for new repositories
+
+A project template repository based on the [template repository](https://github.com/cncf/project-template) by the [Cloud Native Computing Foundation](https://github.com/cncf).
+
+## Configure actions
+
+For the `auto-merge` workflow, you need to add a personal access token to the repository. For more details, see the [documentation for the action](https://github.com/mdn/workflows#auto-merge).
+
+## License
+
+This project is licensed under [Apache license 2.0](LICENSE).
+
+## Conduct
+
+We follow the Mozilla [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Add 'how to use the template' in here, eg:

Copy README-template.md and edit for each repo